src/txn/txn_rollback_to_stable.c

```diff
@@ -613,7 +904,8 @@ __rollback_abort_col_ondisk_kv(WT_SESSION_IMPL *session, WT_REF *ref, WT_COL *ci
           __wt_timestamp_to_string(rollback_timestamp, ts_string[2]));
         if (!F_ISSET(S2C(session), WT_CONN_IN_MEMORY))
             /* Allocate tombstone and calls function add update to head of insert list. */
-            return (__rollback_col_ondisk_fixup_key(session, ref, rollback_timestamp, true, recno));
+            return (
+              __rollback_col_ondisk_fixup_key(session, ref, cip, rollback_timestamp, true, recno));
```

Comment:
Need to add the call to the function `__rollback_col_modify` to let the restored update to be inserted into the update list.

Reply:
__rollback_col_modify is called inside this function __rollback_col_ondisk_fixup_key once it either finds an update in the history store or allocated a tombstone update, does it need to be called here too?

Reply:
I added this to the end of the function __rollback_abort_col_ondisk_kv. Is that correct?